### DISCUSSION TITLE
2023/12/23/advent/

### DISCUSSION THREAD

**Discussion by giscus** [BonyChops/blog#34] [Comment]
# 2023/12/23/advent/

こちらはklis（筑波大学情報学群知識情報・図書館学類） Advent Calendar 2023と長野高専 Advent Calendar 2023 23 日目の記事です！

https://blog.b7s.dev/2023/12/23/advent/

**Comment by RenNishino** [comment]
いい1年だと思いました．出会ってくれてありがとう
